build/azDevOps/azure/air-infrastructure-data-vars-template.yml

```diff
@@ -1,14 +1,14 @@
 variables:
   - name: region
-    value: { { .Project.Cloud.Region } }
+    value: "{ { .Project.Cloud.Region } }"
```

Comment:
@Trishisingh , @satenderrathee - quotes were added to these values in this file and `build/azDevOps/azure/network/air-infrastructure-data-network-vars-template.yml` in order to address yaml linting errors. Please can you verify this won't cause issues?

Reply:
Looks good to me